src/person.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'nameable'
+require_relative 'decorator'
+# Public: Various methods useful for performing checking on people in a library.
+class Person < Nameable
+  # Public: Gets/Sets the String name or age of the person.
+  attr_accessor :name, :age
+
+  # Public: Returns the Integer age of the person.
+  attr_reader :id
+
+  # Public: initialize a person.
+  #
+  # id - An Integer randomly generated.
+  # name - A String naming the person.
+  # age - An Integer that describes the person's age.
+  # parent_permition - A Boolean that describes parent permition status.
+  def initialize(age, name = 'Unknown', parent_permition: true)
+    super()
+    @id = rand(10_000)
+    @name = name
+    @age = age
+    @parent_permition = parent_permition
+  end

```

Comment:
- [ ] Kindly initialize an empty `rentals` array here as you have done in the **Book** class.

Reply:
resolved 👍